packages/client/src/helpers/client/interceptors/authentication/types/TokenManagerOptions.types.ts

```diff
@@ -0,0 +1,59 @@
+import type { AxiosRequestConfig } from 'axios';
+import type { ITokenData } from '../token-providers/types/TokenData.types';
+
+export interface OptionsStorageProvider {
+  getItem(key: string): Promise<string | null> | string | null;
```

Comment:
All methods from OptionsStorageProvider should deal with Promises only like this:

```js
getItem(key: string): Promise<string | null>;
setItem(key: string, value: string): Promise<void>;
removeItem(key: string): Promise<void>;
```

Reply:
done.